content/get_start/UseCase_Workflows/revit.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "PRM for Revit"
+date: 2022-09-21T15:00:28-07:00
+weight: 251
+draft: false
+pre: "<b>- </b>"
+---
+
+{{<line_break>}}
+
+#### A. Use Case 1: Application of the PRM measure to a user-model created in Revit
+
+This use case includes translatiing a BIM model created in Revit to a BEM model and applying PRM measure to automatically generate a baseline and a proposed model based on the PRM rules. 
```

Comment:
Typo:
Translating

let reverse the order say:
a proposed and one or four baseline models

Reply:
Done!